en/_posts/2023-03-17-top-tasks.md

```diff
@@ -0,0 +1,94 @@
+---
+css:
+- ../../../css/blog.css
+- https://use.fontawesome.com/releases/v5.15.4/css/all.css
```

Comment:
Should we move this in the _config.yml in a different PR? If not could we move the css property after the altLangPage property to keep the Frontmatter in alphabetical order.

Reply:
I just copied that path from the previous blog.  Not good style, but got the job done.  Yeah, we should fix that, fore sure.